docs/quarto/index.qmd

```diff
@@ -4,24 +4,37 @@ author: "Nicklas Hafiz"
 date: 12-17-2023
 ---
 
-This website was created using Quarto. It is hosted on GitHub. In this chapter we will look at the underlying structure of this website, and how to update it. 
+This website was created using Quarto. It is hosted on GitHub. In this chapter we will look at the underlying structure of this website, and how to update it.
 
 ## Setup
-This website uses [`renv`](../../posts/r_sig/23_11_06_renv/index.qmd) to keep the project specific package library up to date. The needed packages are recorded in the `.lock`-file, but not uploaded to GitHub. So the *first thing you need to do* is to install the necessary packages into your local project library:
+
+1.  As the website is hosted on GitHub, you can just clone the [repository](https://github.com/iqb-research/IQB-Methods).
+2.  Open the RStudio project with RStudio.
+3.  This website uses [`renv`](../../posts/r_sig/23_11_06_renv/index.qmd) to keep the project specific package library up to date. The needed packages are recorded in the `.lock`-file, but not uploaded to GitHub. So the **first thing you need to do** is to install the necessary packages into your local project library:
 
 ```{r, eval = FALSE}
 # install.packages("renv")
 renv::restore()
 ```
 
+If your files need their own packages, just install them like you would normally do with either:
+
+```{r, eval = FALSE}
+install.packages("eatGADS")
+renv::install("eatGADS")
+```
+
+You need to do this even if you have them already installed on your PC locally, because [`renv`](../../posts/r_sig/23_11_06_renv/index.qmd) uses a project specific library.
+
 ## File structure
-Basically you just need to know where to put your files. Everything else will be taken care of by pushing to GitHub.
-Files can go into one of two folders: `docs` or `posts`. `docs` contains most of the tutorial files: They are structured into sub folders, like `eatPackages` or `R`. Here the quarto-files can be found that contain the actual website content. Just edit them or add new ones. Just make sure they are quarto-files with the `.qmd` ending. 
+
+Basically you just need to know where to put your files. Everything else will be taken care of by pushing to GitHub. Files can go into one of two folders: `docs` or `posts`. `docs` contains most of the tutorial files: They are structured into sub folders, like `eatPackages` or `R`. Here the quarto-files can be found that contain the actual website content. Edit them or add new ones. Just make sure they are quarto-files with the `.qmd` ending.
```

Comment:
Ich würde "just" sparsamer benutzen. "**Just** make sure they are quarto-files with the `.qmd` ending." -> "Make sure they are quarto-files with the `.qmd` ending."